app/assets/json/translations/react/es.json

```diff
@@ -1017,6 +1019,11 @@
   "listings.lotteryPreference.Treasure Island Resident (TIR) Preference.title": "Preferencia de residente de Treasure Island (TIR)",
   "listings.lotteryPreference.allRemainingUnits": "Todas las unidades restantes",
   "listings.lotteryPreference.remainingUnitsAfterPreferenceConsideration": "Después de que se haya tenido en cuenta a todos los titulares preferenciales, las unidades restantes estarán disponibles para los solicitantes calificados en el orden del sorteo.",
+  "listings.lotteryPreference.sfusd.allOtherEmployees.shortTitle": "Todos los demás empleados, SFUSD",
+  "listings.lotteryPreference.sfusd.allOtherEmployees.title": "Todos los demás empleados, Distrito Unificado de Escuelas de San Francisco",
+  "listings.lotteryPreference.sfusd.educators.shortTitle": "Personal docente, SFUSD",
+  "listings.lotteryPreference.sfusd.educators.title": "Personal docente, Distrito Unificado de Escuelas de San Francisco",
+  "listings.lotteryPreference.sfusd.generalPublic.shortTitle": "Público general",
```

Comment:
Going to https://dahlia-webapp-pr-2195.herokuapp.com/listings/a0W4U00000NlolaUAB and entering code 01425245 (following the screenshot on the doc) I don't see this but rather 
<img width="594" alt="Screenshot 2024-06-17 at 9 20 06 AM" src="https://github.com/SFDigitalServices/sf-dahlia-web/assets/36685920/8a4c08c6-0f8b-4fe8-b858-054904a8eed2">

Am I doing something wrong?

Reply:
From the Jira ticket (and also mentioned in the translation doc):
> Some translations only apply to SCV 2 listings, which do not exist in Full yet.

You would have to create the new SCV 2 type of listing in Salesforce. It turns out that there will be new changes to SCV 2, so I think we can ignore this translation for now. I included it for the sake of completeness.

Reply:
Sounds good! Thank you for clarifying!